episodes/12-virtual-environments.md

```diff
@@ -526,10 +571,10 @@ They can then replicate your environment
 and install all the necessary packages from the project root as follows:
 
 ```bash
-(venv) $ python3 -m pip install -r requirements.txt
+(venv) $ python3 -m pip install -r requirements.txt --editable .
```

Comment:
```suggestion
(venv) $ python3 -m pip install -r requirements.txt --exclude-editable .
```
Right?

Reply:
This doesn't make all of the dependencies editable, only those after the `--editable` flag. Everything within `requirements.txt` will be installed "normally" as long as it isn't pre-pended by `-e`. See https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-e